### ISSUE TITLE
Timing of tasks

### ISSUE THREAD

**Issue by Patosga** [klaussinani/taskbook#135]
It would be great to have some sort of task time tracking functionality. 

For example, as a task goes into in-progress there could be some sort of timer which would count the amount of time the task stayed in the in-progress state. Then, the tasks could have a optional time report associated with them.

I've a huge interest in this. However, I don't know how or if it deflects from the original idea of the tool.

Cheers.

**Comment by SonGokussj4** [comment]
Yeah. I was thinking the same thing. And I think it won't be so hard to code. Just some timer when I apply "working on" and stop when deactivated.

Is it possible?

I discovered TB 2 weeks ago and I LOVE it.
There are just some few possible additions that would make it even better. I like the simplicity you achieved :-)

**Comment by allexiusw** [comment]
I think this is neccessary to start, pause and end time to a task, this is a normal flow in a task.
